index.html

```diff
@@ -6,7 +6,34 @@
     <link rel="stylesheet" type="text/css" href="./styles.css">
   </head>
   <body>
+    <section class="sidebar">
+      <h1>IdeaBox</h1>
+      <h2>Filter Starred Ideas</h2>
+      <button type="button" class="show-favorite">Show Starred Ideas</button>
+    </section>
+    <section class="idea-form">
+      <label for="title-input">Title</label>
+      <input type="text" class="title-input" name="title-input">
+      <label for="body-input">Body</label>
+      <input type="text" class="body-input" name="body-input">
+      <button type="button" name="save-button" class="save-button">Save</button>
+      <input type="text" class="search-idea" placeholder="Search ideas">
```

Comment:
The class name of this element seems confusing. "searchIdeas" would be a great name for a function, but seems like a confusing name for an input element.